src/test/java/com/jcabi/jdbc/JdbcSessionITCase.java

```diff
@@ -91,6 +99,93 @@ public void changesTransactionIsolationLevel() throws Exception {
     }
 
     /**
+     * JdbcSession can run a function (stored procedure) with
+     * output parameters.
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void callsFunctionWithOutParam() throws Exception {
+        final DataSource source = JdbcSessionITCase.source();
+        new JdbcSession(source).autocommit(false).sql(
+            "CREATE TABLE IF NOT EXISTS users (name VARCHAR(50))"
+        ).execute().sql("INSERT INTO users (name) VALUES (?)")
+        .set("Jeff Charles").execute().sql(
+            StringUtils.join(
+                "CREATE OR REPLACE FUNCTION fetchUser(username OUT text,",
+                " day OUT date)",
+                " AS $$ BEGIN SELECT name, CURRENT_DATE INTO username, day",
+                " FROM  users; END; $$ LANGUAGE plpgsql;"
+            )
+        ).execute().commit();
+        final Object[] result = new JdbcSession(source)
+            .sql("{call fetchUser(?, ?)}")
+            .prepare(
+                new Preparation() {
+                    @Override
+                    public void
+                        prepare(final PreparedStatement stmt)
+                        throws SQLException {
+                            final CallableStatement cstmt =
+                                (CallableStatement) stmt;
+                            cstmt.registerOutParameter(1, Types.VARCHAR);
+                            cstmt.registerOutParameter(2, Types.DATE);
+                    }
+                }
+             )
+            .call(new StoredProcedureOutcome<Object[]>(2));
+        MatcherAssert.assertThat(result.length, Matchers.is(2));
+        MatcherAssert.assertThat(
+            result[0].toString(),
+            Matchers.containsString("Charles")
+        );
+        MatcherAssert.assertThat(
+            (Date) result[1],
+            Matchers.notNullValue()
+        );
+    }
+
+    /**
+     * JdbcSession can run a function (stored procedure) with
+     * input and output parameters.
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void callsFunctionWithInOutParam() throws Exception {
+        final DataSource source = JdbcSessionITCase.source();
+        new JdbcSession(source).autocommit(false).sql(
+            "CREATE TABLE IF NOT EXISTS usersid (id INTEGER, name VARCHAR(50))"
```

Comment:
@mkordas no, I meant ``usersid`` because its usernames and ids, not only ids of the users